Demo/Demo/ViewController.swift

```diff
@@ -29,13 +30,30 @@ class ViewController: UIViewController {
     }
 
     func displayDemoFeatureViewController() {
-        let demoViewControllerType = DemoSettings.demoType.viewController
-        let demoViewController = demoViewControllerType.init()
-        demoViewController.modalPresentationStyle = .fullScreen
-
-        navigationItem.title = DemoSettings.demoType.rawValue.uppercased()
-
-        addChild(demoViewController)
-        view.addSubview(demoViewController.view)
+        switch DemoSettings.demoUIFramework {
+        case .uikit:
+            let demoViewController = DemoSettings.demoType.viewController
+            demoViewController.modalPresentationStyle = .fullScreen
+
+            navigationItem.title = DemoSettings.demoType.rawValue.uppercased()
+
+            addChild(demoViewController)
+            view.addSubview(demoViewController.view)
+        case .swiftui:
+            let swiftUIController = UIHostingController(rootView: SwiftUICardDemo(baseViewModel: BaseViewModel()))
```

Comment:
Should we also add something like this in `DemoType.swift` so we can get corresponding swiftUI content view similar to how we get view controller for `uikit` case above:
```
var swiftUIView: Content {
   switch self {
      case .card:
          ...
      case .paypal:
          ...
   }
}
```

Reply:
I think we _could_ but currently our PayPalButton(s) aren't representable in SwiftUI. We should be able to use [`UIViewRepresentable`](https://developer.apple.com/documentation/swiftui/uiviewrepresentable) in a similar manner to how this PR uses [`UIViewControllerRepresentable`](https://developer.apple.com/documentation/swiftui/uiviewcontrollerrepresentable) to create a public wrapper in our PayPal interface to allow our buttons to be represented easily in SwiftUI, but that'd be at the SDK level, so I didn't want to make that change quite yet without chatting as a team.

That to say, I think we should and then can make the above change as well as add a SwiftUI View for the PayPal case,  but for now the PayPal case just takes you to our UIKit view. Since this will require some work and a team chat, thoughts on punting on this for now - with the hopes that this can be a follow on story to make this change at the SDK level and add out buttons natively via SwiftUI to the demo?

Reply:
ah got it!
what do you think of just using an empty `UIView()` or something similar for PayPal SwiftUI case for now, so we won't show card demo when PayPal + SwiftUI is selected?

Reply:
Sure - that makes sense for now, will add.